src/main/java/com/kiwigrid/helm/maven/plugin/AbstractHelmMojo.java

```diff
@@ -49,6 +49,9 @@
 	@Parameter(property = "helm.chartVersion", required = true)
 	private String chartVersion;
 
+	@Parameter(property = "helm.AppVersion", required = true)
```

Comment:
Make it optional.
Fix property name, should be `helm.appVersion`

Reply:
Fixed

Reply:
Please adjust property name.